### PR TITLE
Docs 2128 tokenization

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

**Review by fermujica — APPROVED** [review]

**Review by Mulesoft-Mysti — APPROVED** [review]
I think our style is to spell out numbers lower than ten.
"force one invalid digit" might be easier to read but I'm not sure of the conventions in this area of the product.

File: modules/ROOT/pages/tokenization-formats.adoc [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 Tokenization formats define how the original format of data coming in is converted to the format you configure when the token is created.  For example, you can define whether to preserve certain digits, if the token should look like the original data source, or whether to tokenize the whole string or partial string.
 
+== Format Preserving Tokenization
+
+Application data validation logic work “as is”. +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization-formats.adoc]: should be "works"
does "as is" need quotes? should it be "as defined" (defined where? in other words "as is" according to what, if not clear from context.)

File: modules/ROOT/pages/tokenization-formats.adoc [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+No downstream application changes needed. Compliance scope reduction (PCI, HIPAA, GDPR)
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization-formats.adoc]: Neither of these are grammatical sentences, but they should either both have periods (.) or neither?

File: modules/ROOT/pages/tokenization.adoc [hunk 3]
```diff
@@ -2,16 +2,33 @@
...
 
-The tokenization service addresses a variety of data security needs:
-
-== Format Preserving Tokenization
-
-Application data validation logic work “as is”. +
-No downstream application changes needed. Compliance scope reduction (PCI, HIPAA, GDPR)
-
-image::index-a21f8.png[index-a21f8]
+The Anypoint Security tokenization service enables you to substitute a sensitive data element with a non-sensitive equivalent. Anypoint tokenization can create Format Preserving Tokens, which means the output tokens will have the same format as the sensitive data input.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc]: No reason to capitalize format-preserving tokens?
Agree with leaving "will" in, in this case.

File: modules/ROOT/pages/tokenization.adoc [hunk 4]
```diff
@@ -2,16 +2,33 @@
...
+
+Examples of sensitive information that are suitable for tokenization protection include:
+
+* Primary Account Number (PAN)
+* Personally Identifiable Information (PII)
+* Protected Health Information (PHI), or any information deemed sensitive.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc]: no period since the other elements don't have one.

File: modules/ROOT/pages/tokenization.adoc [hunk 5]
```diff
@@ -2,16 +2,33 @@
...
+
+Anypoint Security tokenization provides the following benefits:
+
+* A vaultless operation, ensuring sensitive data is not stored in the tokenization deployment:
+** No data growth based on the number of transactions processed or tokens issued, thus removing data-vault management issues.
+* Increased security for sensitive information thereby reducing your “Sensitive Data Zones”:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc]: remove quotes and caps for sensitive data zones

File: modules/ROOT/pages/tokenization.adoc [hunk 6]
```diff
@@ -2,16 +2,33 @@
...
+** Reduces or removes the number of systems and applications that process, store, or transmit sensitive information from sensitive data scopes.
+** Negates the chances of a data breach from auxiliary applications and data stores, as the sensitive information stored in them is replaced by tokens.
+** Increases defense in depth for sensitive data and least privilege. Authorization is required to the tokenization service to detokenize the sensitive data back to its actual value. Possession of the token alone provides no extrinsic or exploitable meaning or value.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc:23]: I don't think extrinsic is helpful here, covered by exploitable.
I'd rewrite "Posssesion of the token alone provides no meaning or value to exploit."

File: modules/ROOT/pages/tokenization.adoc [hunk 7]
```diff
@@ -2,16 +2,33 @@
...
+* Flexibility:
+** Ability to choose a tokenization “scheme” appropriate for your business using inbuilt data domains (PAN, SSN, email, decimal, ascii and so on).
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc:25]: built-in not inbuilt, and ASCII is an acronym, and remove ", and so on" (I put "such as" or "including" at the beginning of incomplete lists)

File: modules/ROOT/pages/tokenization.adoc [hunk 8]
```diff
@@ -2,16 +2,33 @@
...
+** Each domain has “configurable” options for preserving portions of the original sensitive data and replacing the rest with token data or replacing the entire information with a token.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc:26]: remove quotes around "configurable"

File: modules/ROOT/pages/tokenization.adoc [hunk 9]
```diff
@@ -2,16 +2,33 @@
...
+** Operates seamlessly across wide area networks and distributed data centers.
+* Format Preserving Tokenization capability ensures that:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc]: I feel like Format Preserving Tokenization capability ensures: should be a new paragraph, not a list item as it's not parallel to the other two top bullets.

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/tokenization.adoc]: Also it should have a hyphen and doesn't need to be capitalized: Format-preserving tokenization ensures:

File: modules/ROOT/pages/tokenization.adoc [hunk 10]
```diff
@@ -2,16 +2,33 @@
...
+** No changes are required to existing enterprise data flows or data stores, as the solution ensures that the tokens generated conform to the existing data structure and validations.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc]: No changes are required to existing enterprise data flows or data stores, as the tokens generated conform.... (don't need "as the solution ensures that")

File: modules/ROOT/pages/tokenization.adoc [hunk 11]
```diff
@@ -2,16 +2,33 @@
...
+** Changes to existing applications are minimized compared to point-to-point encryption, which requires data protection implementation at each “hop”.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc]: period inside quotes.

File: modules/ROOT/pages/tokenization.adoc [hunk 12]
```diff
@@ -2,16 +2,33 @@
...
+* Performance:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc]: I'd move this above the set of bullets that I suggested have a new paragraph lead in, if you take that suggestion.

File: modules/ROOT/pages/tokenization.adoc (hunk 9 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/tokenization.adoc]: Or, you can leave it here but make it a noun phrase, not a sentence, to be parallel with the other top-level bullets, something like "Token Format Preservation" or if it's standard industry term, leave it Format-Preserving Tokenization

File: modules/ROOT/pages/tokenization.adoc [hunk 13]
```diff
@@ -2,16 +2,33 @@
...
+** High-performance operation ensures low-latency processing.
+** Performance scales linearly on each node with vertical and horizontal scaling for vaultless tokenization provided by the Anypoint Platform.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/tokenization.adoc:30]: I don't understand how two-dimensional scaling can be linear scaling, and I don't really understand what this sentence means. do customers also have valutless tokenization not provided by Anypoint Platform, or is that an unnecessary qualifier? Each node of what? What are the vertical and horizontal axes being referenced?

**Review by Mulesoft-Mysti — APPROVED** [review]
Line 31 okay to publish, but I'd follow up (unless my lack of knowledge of this area is the problem). Everything else looks great!

File: modules/ROOT/pages/tokenization-example.adoc [hunk 14]
```diff
@@ -0,0 +1,262 @@
...
+The following is an example of bad tokenization JSON data stopped by Api Gateway protection:
+
+`curl ­v ­-k ­­--resolve token2mule.ic.intel.com:443:10.230.36.230 \https://token2mule.ic.intel.com/tb/v1/detokenization ­-X POST -­H "Content­type: application/json" ­­data '[{{{}{{{}]]"data": "597­74­8102­­­­­­­­sdsdsdsdsdsdsdsds", "format": "ssndemo"}]' ­-k ­­--user test:test
+
+HTTP/1.1 400 Bad Request
+{ "errorcode": 1140, "message": "Error while parsing json [line 1 char 3, byte-offset 2]: Expected member name"}``
+
+== Logs
+
+You can use the tokenization summary message that is returned in the logs to determine traffic counts. To retrieve the logs, follow the instructions in the xref:configure-log-forwarding.adoc[Runtime Fabric documentation] for configuring log forwarding.
```
Comment (sleslie23) [inline modules/ROOT/pages/tokenization-example.adoc:223]: @valkyrie69 Here is the broken xref.

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/tokenization-example.adoc:223]: Sorry @valkyrie69 I should have seen that!

**Pull request merged**